### PR TITLE
i18n(fr): update `api-reference.mdx` and `guides/view-transitions.mdx`

### PR DESCRIPTION
**Pull request by ArmandPhilippot**
#### Description (required)

* Add Actions section in `api-reference.mdx` (see #9224 and #9254)
* Add View Transitions section in `api-reference.mdx` (see #9174)
* Add changes from #9261 to Actions `handler` in `api-reference.mdx`
* Add `Building a custom swap function` section in `guides/view-transitions.mdx` (see #9224, added to this PR to avoid a broken link)

#### Related issues & labels (optional)

- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/fr/reference/api-reference.mdx [hunk 1]
```diff
@@ -2075,6 +2137,462 @@ export const onRequest = defineMiddleware(async (context, next) => {
...
+
+### `getFallback`
+
+<p>
+
+**Type :** `() => 'none' | 'animate' | 'swap'`<br />
+<Since v="3.6.0" />
+</p>
+
+Renvoie la stratégie de secours à utiliser dans les navigateurs qui ne prennent pas en charge les transitions de vue.
```
Comment (thomasbnt) [inline src/content/docs/fr/reference/api-reference.mdx:2255]:
```suggestion
Renvoie la stratégie de secours à utiliser dans les navigateurs qui ne prennent pas en charge les Transitions de Vue.
```

File: src/content/docs/fr/reference/api-reference.mdx [hunk 2]
```diff
@@ -2075,6 +2137,462 @@ export const onRequest = defineMiddleware(async (context, next) => {
...
+
+Voir le guide sur la [Gestion de solution de secours](/fr/guides/view-transitions/#gestion-de-solution-de-secours) pour savoir comment choisir et configurer le comportement de repli.
+
+
+### Événement `astro:before-preparation`
+
+Un événement déclenché au début d'une navigation à l'aide de transitions de vue. Cet événement se produit avant toute demande et tout changement d'état du navigateur.
```
Comment (thomasbnt) [inline src/content/docs/fr/reference/api-reference.mdx:2262]:
```suggestion
Un événement déclenché au début d'une navigation à l'aide de Transitions de Vue. Cet événement se produit avant toute demande et tout changement d'état du navigateur.
```

File: src/content/docs/fr/reference/api-reference.mdx [hunk 3]
```diff
@@ -2075,6 +2137,462 @@ export const onRequest = defineMiddleware(async (context, next) => {
...
+
+L'entrée de l'historique et la position de défilement ont déjà été mises à jour lorsque cet événement est déclenché.
+
+### Événement `astro:page-load`
+
+Un événement envoyé une fois le chargement d'une page terminé, qu'il s'agisse d'une navigation utilisant des transitions de vue ou native du navigateur.
+
+Lorsque les transitions de vue sont activées sur la page, le code qui s'exécuterait normalement sur `DOMContentLoaded` doit être modifié pour s'exécuter sur cet événement.
+
+### Attributs des événements de transitions de vue
```
Comment (thomasbnt) [inline src/content/docs/fr/reference/api-reference.mdx:2314]:
```suggestion
### Attributs des événements de Transitions de Vue
```

**Comment by ArmandPhilippot** [comment]
Thanks @thomasbnt! Some are not capitalized because they are not (and [should not](https://github.com/withastro/docs/pull/9174#discussion_r1736626975) be) capitalized in the English documentation. In summary, if "view transitions" refers to the Astro API, it should be uppercase, but otherwise it should be lowercase. So I applied the same rule for `transitions de vue`.

I'll check that I haven't forgotten some capital letters where it applies, but for the rest, I'm not sure that the syntax needs to be updated.

**Comment by thomasbnt** [comment]
> Thanks @thomasbnt! Some are not capitalized because they are not (and [should not](https://github.com/withastro/docs/pull/9174#discussion_r1736626975) be) capitalized in the English documentation. In summary, if "view transitions" refers to the Astro API, it should be uppercase, but otherwise it should be lowercase. So I applied the same rule for `transitions de vue`.
> 
> I'll check that I haven't forgotten some capital letters where it applies, but for the rest, I'm not sure that the syntax needs to be updated.

Okay! Then, LGTM!

**Review by thomasbnt — APPROVED** [review]
LGTM!

File: src/content/docs/fr/reference/api-reference.mdx [hunk 4]
```diff
@@ -2075,6 +2137,462 @@ export const onRequest = defineMiddleware(async (context, next) => {
...
+Implémentation de la phase suivante dans la navigation (chargement de la page suivante). Cette implémentation peut être surchargée pour ajouter un comportement supplémentaire.
+
+#### `viewTransition`
+
+<p>
+
+**Type :** [`ViewTransition`](https://developer.mozilla.org/en-US/docs/Web/API/ViewTransition)
+</p>
+
+L'objet de transition de vue utilisé dans cette navigation. Sur les navigateurs qui ne prennent pas en charge l'[API de transitions de vue](https://developer.mozilla.org/en-US/docs/Web/API/View_Transitions_API), il s'agit d'un objet implémentant la même API pour plus de commodité, mais sans l'intégration DOM.
```
[the remark this answers is not in the document]
Comment (ArmandPhilippot) [reply src/content/docs/fr/reference/api-reference.mdx:2421]:
```suggestion
L'objet de transition de vue utilisé dans cette navigation. Sur les navigateurs qui ne prennent pas en charge l'[API de Transitions de Vue](https://developer.mozilla.org/en-US/docs/Web/API/View_Transitions_API), il s'agit d'un objet implémentant la même API pour plus de commodité, mais sans l'intégration DOM.
```

**Comment by ArmandPhilippot** [comment]
Some of your suggestions was legit, this is fixed now. Thanks @thomasbnt!

**Review by yanthomasdev — APPROVED** [review]
LGTM! Good job peeps!

**Pull request merged**